docs/attributes-registry/user-agent.md

```diff
@@ -8,5 +8,12 @@
 <!-- semconv registry.user_agent(omit_requirement_level) -->
 | Attribute  | Type | Description  | Examples  |
 |---|---|---|---|
+| `user_agent.device.name` | string | Name of the device extracted from original | `iPhone` |
+| `user_agent.name` | string | Name of the user-agent extracted from original. Usually refers to the browser's name [1] | `Safari` |
 | `user_agent.original` | string | ![Stable](https://img.shields.io/badge/-stable-lightgreen)<br>Value of the [HTTP User-Agent](https://www.rfc-editor.org/rfc/rfc9110.html#field.user-agent) header sent by the client. | `CERN-LineMode/2.15 libwww/2.17b3`; `Mozilla/5.0 (iPhone; CPU iPhone OS 14_7_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.1.2 Mobile/15E148 Safari/604.1` |
+| `user_agent.version` | string | Version of the user-agent extracted from original. Usually refers to the browser's version [2] | `14.1.2` |
```

Comment:
This has sort of already been defined (see the above browser namespace), the approach taken by using the "brands" is because browsers have "frozen" the UserAgent and will not necessarily identify the "true" name (for the browser). Currently, the major version number is still Updated but it will not included the full version number in the UA.

Reply:
In this PR we are extracting data provided in `user_agent.original` into separate fields so user could use already extracted data for their needs. We relaying on these fields for example in around 70 different integrations in elastic security product. I think these are different means, the ones we are extracting from browser itself and the ones used here

Reply:
My main concerns here are the following:

1. User-Agent gets used for any HTTP traffic, including microservices.  This definition is both generic AND tied to browser-based HTTP traffic.  I'd love of if this field worked for, e.g. OTLP-HTTP (or other gRPC-HTTP) traffic, CURL and browsers.  
2. For microservices (like gRPC, or OTLP-HTTP specifically) there can multiple versions (e.g. the version of gRPC-go + the version of the otel collector used).  Should we allow for multiple versions here *or* somehow specify that `user_agent.version` + `user_agent.name` is somehow the "dominant" software?


I think for browsers, it's clear what to do with these fields and I have no problem with the definitions as-is for that use case.  I just want to figure out (a) do we allow the microservice-http use case with these fields and if so (b) how would you alter the description to make that work?